tests/src/test/java/org/wso2/micro/gateway/tests/grpc/GrpcServer.java

```diff
@@ -31,7 +31,7 @@
 
     public void start() throws IOException {
         /* The port on which the server should run */
-        int port = 50051;
+        int port = 50075;
```

Comment:
Any reason to change the port? If there is an specific reason, better if you can put that in the PR description. It might be useful later.

Reply:
There seems to be port conflict with mac. Its better to use a different port